src/global/components/MenuDropdown.tsx

```diff
@@ -0,0 +1,25 @@
+import { useNavigate } from "react-router-dom";
+import { useRecoilState } from "recoil";
+import { menuState } from "../state/navbarState";
+import { Path } from "../types/navigationType";
+
+export default function MenuDropdown(props: Path) {
+  const navigate = useNavigate();
+  const [isMenuOpen, setIsMenuOpen] = useRecoilState(menuState);
+
+  const handleNavigate = () => {
+    setIsMenuOpen(false);
+    navigate(props.path);
+  };
+
+  return (
+    <li className="h-[70px] md:h-[50px] hover:bg-light-gray-element-color transition ease-in-out delay-75 border-b border-light-gray-background-color !-z-1">
+      <button
```

Comment:
It's bad practice to use `button` for navigation, you should always use `link`. Then you can check if the route has changed to close the menu

Reply:
Added Link instead of button.